app/frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "vue-router": "^3.0.1",
     "vue-select": "^2.5.0",
     "vue-smoothscroll": "^0.1.1",
+    "vue2-leaflet": "^1.0.2",
     "vuex": "^3.0.1"
```

Comment:
You're also going to need to pull in the esri leaflet package to load in the esri layers.